packages/abap-deploy-config-inquirer/src/translations/abap-deploy-config-inquirer.i18n.json

```
@@ -129,19 +129,26 @@
             "appNameInvalid": "Application name invalid",
             "appNameInvalidLength": "Application name limited to 15 characters, entered {{length}}",
             "appNameInvalidPrefix": "Application name needs to be prefixed with [{{prefix}}]",
-            "abapAppDescLength": "Deployment Description should not exceed 60 characters",
+            "abapAppDescLength": "Deployment Description must not exceed 60 characters",
             "abapTransportNumRequired": "Provide a transport request for your application",
             "abapPackageWarn": "Provide a package",
-            "abapInvalidNamespace": "Namespace invalid as there are too many forward slashes",
-            "abapInvalidNamespaceLength": "Namespace limited to 10 characters, entered {{length}}",
+            "abapPackageInvalidFormat": "The package name does not match the expected format. The expected format is `/<namespace>/<package_name>`  or a package name. If a namespace is not specified, the package name must start with '$', 'Z', 'Y', or 'SAP'.",
+            "abapPackageStartingPrefix": "The package must start with either a namespace, '$', 'Z', 'Y', or 'SAP'",
+            "abapInvalidNamespace": "The namespace is invalid because there are too many forward slashes.",
+            "abapInvalidNamespaceLength": "The namespace is limited to 10 characters, but you entered {{length}}.",
```

```suggestion
            "abapInvalidNamespaceLength": "The namespace is limited to 10 characters. You entered {{length}}.",
```
Reapplying suggestion because it says it is outdated.